charts/vsphere-csi/v2.1.0/values.yaml

```diff
@@ -1,29 +1,83 @@
-vCenter:
-  host: ""
-  datacenters: ""
-  port: 443
-  insecureFlag: true
-  username: ""
-  password: ""
-  configSecret:
-    name: "vsphere-config-secret"
-    generate: true
-    configTemplate: |
-      [Global]
-      cluster-id = {{ uuidv4 | quote }}
-      user = {{ .Values.vCenter.username | quote }}
-      password = {{ .Values.vCenter.password | quote }}
-      port = {{ .Values.vCenter.port | quote }}
-      insecure-flag = {{ .Values.vCenter.insecureFlag | quote }}
-
-      [VirtualCenter {{ .Values.vCenter.host | quote }}]
-      datacenters = {{ .Values.vCenter.datacenters | quote }}
+csiController:
+  vsphereConfig:
+    host: ""
+    datacenters: ""
+    port: 443
+    insecureFlag: true
+    username: ""
+    password: ""
+    secret:
+      name: "vsphere-config-secret"
+      generate: true
+      configTemplate: |
+        [Global]
+        cluster-id = {{ uuidv4 | quote }}
+        user = {{ .Values.csiController.vsphereConfig.username | quote }}
+        password = {{ .Values.csiController.vsphereConfig.password | quote }}
+        port = {{ .Values.csiController.vsphereConfig.port | quote }}
+        insecure-flag = {{ .Values.csiController.vsphereConfig.insecureFlag | quote }}
+
+        [VirtualCenter {{ .Values.csiController.vsphereConfig.host | quote }}]
+        datacenters = {{ .Values.csiController.vsphereConfig.datacenters | quote }}
+
+  image:
+    repository: gcr.io/cloud-provider-vsphere/csi/release/driver
```

Comment:
These images will be mirrored. Kept the default so that QA can test asap.

Reply:
images have been mirrored https://github.com/rancher/image-mirror/pull/78

Reply:
Same for everything below